src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from './components/Header';
+import Router from './components/Router';
+
+export function App() {
+  return (
+    <>
+      <Header />
+      <Router />
+    </>
```

Comment:
Instead of writing Header componenet above Router, I suggest you to create Layout component in components folder.
In your case: 

```js
const Layout =({children}) =>{
    return(
        <>
            <Header />
            <div>
                { children }
            </div>
        </>
    )
}

```

in App component you can define routes

``` js
return (
    <Routes>
        <Route path="/" element={<Layout />}>
            <Route index element={<HomePage />} />
            <Route path="about" element={<AboutPage />} />
        </Route>
        <Route path="*" element={<NotFound />}/>
    </Routes>
)
```